grails-app/controllers/au/org/ala/merit/OrganisationController.groovy

```diff
@@ -12,7 +14,7 @@ class OrganisationController {
     static allowedMethods = [ajaxDelete: "POST", delete:"POST", ajaxUpdate: "POST", prepopulateAbn:"GET"]
 
     def organisationService, searchService, documentService, userService, roleService, commonService, webService
-    def activityService, metadataService, projectService, excelImportService, reportService, pdfConverterService, authService
+    def activityService, metadataService, projectService, excelImportService, reportService, pdfConverterService, authService, managementUnitService
```

Comment:
The organisation controller shouldn't have a dependency on the managementUnitService.